docs/conf.py

```diff
@@ -12,6 +12,8 @@
 #
 # import os
 # import sys
+# sys.path.insert(0, os.path.abspath('.'))
```

Comment:
Should we remove these lines?

Reply:
I want to keep these. 

Apparently, sphinx can have some import issues and this is one of the possible solutions for it. 

I do not know why we did not get those issues. Just in case we end up getting it in the future, I left those in.